Mathlib/Logic/Equiv/MfldSimpsAttr.lean

```diff
@@ -0,0 +1,37 @@
+import Lean
+
+open Lean Meta
+
+initialize mfldSetTacExt : SimpExtension ←
```

Comment:
@gebner, the docBlame linter is complaining about `mfldSetTacExt` here, but I don't think this is helpful. Not sure the best approach -- tweak the linter, or try to add `@[nolint docBlame]` automatically, or ...?

Reply:
This is just like a definition, why shouldn't it have a docstring?

```lean
/-- The environment extension for the `mfld_simps` simp set. -/
```

I'd be more concerned about the duplication of the description.  If you're motivated, you could add a macro that automates this whole file:
```lean
/-- The simpset `mfld_simps` records several simp lemmas that are
especially useful in manifolds. It is a subset of the whole set of simp lemmas, but it makes it
possible to have quicker proofs (when used with `squeeze_simp` or `simp only`) while retaining
readability.
The typical use case is the following, in a file on manifolds:
If `simp [foo, bar]` is slow, replace it with `squeeze_simp [foo, bar, mfld_simps]` and paste
its output. The list of lemmas should be reasonable (contrary to the output of
`squeeze_simp [foo, bar]` which might contain tens of lemmas), and the outcome should be quick
enough. -/
simp_set mfld_simps
```

Reply:
@gebner Note that this and other PRs are blocked by the fact that you *can't* put docstrings on `initialize`. This is being tracked in leanprover/lean4#1324, and hopefully the fix will be merged today or tomorrow.

Reply:
Ah, damn. :cry:   #337

Reply:
Okay, now that leanprover/lean4#1324 has landed, I've added the doc-string here.